### PR TITLE
docs: embed streamlit operation matrix app to docs

### PR DESCRIPTION
**Pull request by cpcloud**
This PR embeds the streamlit backend operation matrix app in our docs.

### CODE REVIEW

**Comment by gforsyth** [comment]
I tried to build this locally to check out how it looks and I'm getting a permission error -- should this work on a local docs build?

**Comment by gforsyth** [comment]
Apparently I forgot what URLs were for a second.

Anyway, I can't access https://ibis-project-ibis-docsbackend-info-app-k0skja.streamlit.app in browser directly, so I think we need to set some permissions somewhere.

**Comment by cpcloud** [comment]
Yep, lemme fix that up.

**Review by gforsyth — APPROVED** [review]
:shipit:

**Comment by jcrist** [comment]
This doesn't seem to render properly on firefox (I'm on version 108.0.1 if that matters). I checked chrome and that looks fine, but on firefox the app window is super tiny.

image

**Comment by cpcloud** [comment]
Hm, lemme see if I can reproduce

File: .streamlit/config.toml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[browser]
+gatherUsageStats = false
+
+[theme]
+base = "dark"
```
Comment (krzysztof-kwitt) [inline .streamlit/config.toml:5]: It seems to me that this file must be in the directory with the application i.e. `docs/backends/app/.streaamlitt/config.toml`

Comment (cpcloud) [reply .streamlit/config.toml:5]: I read the docs as saying it must be in the directory the application is run from, which I assumed to be the repo:

image

**Pull request merged**